packages/libs/eda/src/lib/map/analysis/hooks/standaloneMapMarkers.tsx

```diff
@@ -316,30 +368,50 @@ export function useStandaloneMapMarkers(
                 },
               ];
 
+        // here's what Bob pointed out
         const count =
           vocabulary != null // if there's an overlay (all expected use cases)
             ? overlayValues.reduce((sum, { count }) => (sum = sum + count), 0)
```

Comment:
just curious, why not just use entityCount always?

Reply:
Dunno, I just left this as it was!